intuita-webview/src/sourceControl/App.tsx

```diff
@@ -28,7 +28,7 @@ const getViewComponent = (view: View) => {
 		case 'createIssue': {
 			return <CreateIssue {...view.viewProps} />;
 		}
-		case 'createPR':
+		case 'createOrUpdatePR':
```

Comment:
how about `upsertPR`? or `upsertPullRequest` (I prefer not to use acronyms if we could replace them with short names).